fastestimator/cli/train.py

```diff
@@ -38,12 +38,20 @@ def _get_estimator(args: Dict[str, Any], unknown: Optional[List[str]]) -> Estima
     if args['hyperparameters_json']:
         hyperparameters = os.path.abspath(args['hyperparameters_json'])
         hyperparameters = json.load(open(hyperparameters, 'r'))
+    warmup = True
```

Comment:
this isn't the right way to add an argument to the CLI. it won't show up in the -h print message. Instead:

parser.add_argument('--no_warmup', help="Disable warmup for this session", action='store_true')

Reply:
done